File: python/HELLO.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  HELLO.py
#  
#  Copyright 2017  <>
#  
#  This program is free software; you can redistribute it and/or modify
#  it under the terms of the GNU General Public License as published by
#  the Free Software Foundation; either version 2 of the License, or
#  (at your option) any later version.
#  
#  This program is distributed in the hope that it will be useful,
#  but WITHOUT ANY WARRANTY; without even the implied warranty of
#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#  GNU General Public License for more details.
#  
#  You should have received a copy of the GNU General Public License
#  along with this program; if not, write to the Free Software
#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
#  MA 02110-1301, USA.
#  
#  
rok = 2017
py = 1991

def parzyste (n):
    ile = list(range(0, n+1, 2))
    print(ile)
    return len(ile)

def main(args):
    return 0

if __name__ == '__main__':
    
    imie = input("Podaj imię ")
    print ("HELLO", imie)
    
    wiek = int(input("Ile masz lat?"))
    print ("Masz", wiek, "lat i urodziles sie w", 2017 - wiek, "roku")
    
    if wiek < (rok - py):
        print ("Jestem starszy")
    elif wiek > (rok - py):
        print ("Jestem mlodszy")
    else:
        print ("Mamy tyle samo lat")
        
    n = int(input("Podaj liczbe: "))
    print("Parzystych:", parzyste(n))
    
    import sys
    sys.exit(main(sys.argv))

File: C++/hello.cpp
/*
 * hello.cpp
 * 
 * Copyright 2017  <>
 * 
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.
 * 
 * 
 */


#include <iostream>
using namespace std;
#define cpp 1983
#define rok 2017

int parzyste(int liczba){
    int i;
    for ( i=0; i <= liczba; i+=2)
        cout << i << " ";
    return i/2;
    }


int main(int argc, char **argv)
{   
    string imie;
    int wiek, liczba;
    cout << "Podaj imie" << endl;
    cin >> imie;
    cout << "HELLO " << imie << endl;
    cout << "Ile masz lat? " << endl;
	cin >> wiek;
    cout << "Urodziles sie w " << 2017 - wiek << " roku" << endl;
    
    if(wiek < rok - cpp)
        cout << "Jestem starszy" << endl;
    else if( wiek > rok-cpp)
        cout << "Jestem mlodszy" << endl;
    else{
        cout << "Mamy tyle samo lat";
    }
    
    cout << "Podaj liczbe naturalna" << endl;
    cin >> liczba;
    cout << "Ilosc liczb parzystych: " << parzyste(liczba) << endl;
    

    return 0;
}

